index.html

```diff
@@ -5007,11 +5006,11 @@ <h3>Proofs (Signatures)</h3>
 
         <p>
 The cryptographic mechanism used to prove that the information in a
-<a>verifiable credential</a> or a <a>verifiable presentation</a> has not been
-tampered with is called a <em>proof</em>. There are many types of
-cryptographic proofs including, but not limited to, digital signatures,
-zero-knowledge proofs, Proofs of Work, and Proofs of Stake. In general, when
-verifying proofs implementations are expected to ensure that:
+<a>verifiable credential</a> or <a>verifiable presentation</a> was not
+tampered with is called a <em>proof</em>. There are many types of cryptographic
+proofs including, but not limited to, digital signatures, zero-knowledge proofs,
+Proofs of Work, and Proofs of Stake. In general, when verifying proofs
```

Comment:
`when verifying proofs` -> `when verifying proofs,`

Reply:
A comma is not specifically required here, but ok.